src/test/java/org/mapdb/io/DataIOTest.java

```diff
@@ -335,4 +366,295 @@ public void testUnpackLong_withInputStream_throws_exception_when_stream_is_empty
                 DataIO.packLongSize(1 << 63));
     }
 
-}
\ No newline at end of file
+    @Test public void testUnpackInt() throws IOException {
```

Comment:
`@Test` should be on its own line (throughout test class)

Reply:
their own tests keep the `@Test` annotation on the same line which is why I changed mine to keep them similar.

Reply:
You're right, I withdraw that comment.